t/plugin/clickhouse-logger.t

```diff
@@ -0,0 +1,222 @@
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "clickhouse-logger": {
+                                "user": "default",
+                                "password": "a",
+                                "database": "default",
+                                "logtable": "t",
+                                "endpoint_addr": "http://127.0.0.1:10420/clickhouse-logger/test"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1982": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/opentracing"
+                }]],
+                [[{
+                    "action":"set",
+                    "node":{
+                        "value":{
+                            "uri":"/opentracing",
+                            "upstream":{
+                                "scheme":"http",
+                                "nodes":{
+                                    "127.0.0.1:1982":1
+                                }
+                            },
+                            "plugins":{
+                                "clickhouse-logger":{
+                                    "batch_max_size":1000,
+                                    "max_retry_count":0,
+                                    "retry_delay":1,
+                                    "ssl_verify":true,
+                                    "endpoint_addr":"http://127.0.0.1:10420/clickhouse-logger/test",
+                                    "password":"a",
+                                    "buffer_duration":60,
+                                    "timeout":3,
+                                    "user":"default",
+                                    "name":"clickhouse-logger",
+                                    "database":"default",
+                                    "logtable":"t",
+                                    "inactive_timeout":5
+                                }
+                            },
+                            "id":"1"
+                        },
+                        "key":"/apisix/routes/1"
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 5: access local server
+--- request
+GET /opentracing
+--- response_body
+opentracing
+--- grep_error_log_out
+"clickhouse body: INSERT INTO t FORMAT JSONEachRow"
+"clickhouse headers: X-ClickHouse-Key:a"
+"clickhouse headers: X-ClickHouse-User:default"
+"clickhouse headers: X-ClickHouse-Database:default"
```

Comment:
I have reconsidered the test.
What about using error_log like https://github.com/apache/apisix/blob/537a8da24d723caa5c85561d36b3b931ce1f1410/t/plugin/loggly.t#L572?

Reply:
It doesn't work.

Reply:
It doesn't work. I don't know why. I think `--- grep_error_log_out`  is easier to use.

Reply:
Use `--- error_log` fialed. https://github.com/apache/apisix/runs/5197064703?check_suite_focus=true#step:14:891